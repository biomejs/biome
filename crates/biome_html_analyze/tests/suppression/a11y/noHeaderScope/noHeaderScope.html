<!-- code that should trigger the rule and show suppression actions -->
<div scope="col"></div>
