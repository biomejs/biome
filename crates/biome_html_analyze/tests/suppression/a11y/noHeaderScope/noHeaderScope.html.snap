---
source: crates/biome_html_analyze/tests/spec_tests.rs
expression: noHeaderScope.html
---
# Input
```html
<!-- code that should trigger the rule and show suppression actions -->
<div scope="col"></div>

```

# Diagnostics
```
noHeaderScope.html:2:6 lint/a11y/noHeaderScope  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Avoid using the scope attribute on elements other than th elements.
  
    1 │ <!-- code that should trigger the rule and show suppression actions -->
  > 2 │ <div scope="col"></div>
      │      ^^^^^^^^^^^
    3 │ 
  
  i The scope attribute is used to associate a data cell with its corresponding header cell in a data table,
                so it should be placed on th elements to provide accessibility to screen readers.
  
  i Follow the links for more information,
                WCAG 1.3.1
                WCAG 4.1.1
  
  i Safe fix: Suppress rule lint/a11y/noHeaderScope for this line.
  
    1 1 │   <!-- code that should trigger the rule and show suppression actions -->
    2   │ - <div·scope="col"></div>
      2 │ + <!--·biome-ignore·lint/a11y/noHeaderScope:·<explanation>·-->
      3 │ + <div·scope="col"></div>
    3 4 │   
  
  i Safe fix: Suppress rule lint/a11y/noHeaderScope for the whole file.
  
    1 1 │   <!-- code that should trigger the rule and show suppression actions -->
    2   │ - <div·scope="col"></div>
      2 │ + <!--·biome-ignore-all·lint/a11y/noHeaderScope:·<explanation>·-->
      3 │ + <div·scope="col"></div>
    3 4 │   
  

```
